### ISSUE TITLE
whoami command

### ISSUE THREAD

**Issue by yakuter** [yakuter/gol#10]
Get the active username command.

**Comment by abdrd** [comment]
May I implement this one ?

**Comment by yakuter** [comment]
Of course. It is on you @betelgeuse-7

**Comment by abdrd** [comment]
Thanks!

**Comment by abdrd** [comment]
Is it okay if I don't use ```github.com/urfave/cli/v2``` package ?

**Comment by abdrd** [comment]
I am planning to implement other commands, and I don't like to use an abstraction for such small applications.

**Comment by yakuter** [comment]
Well this cli package is the skeleton of our CLI application "gol". So I expect you to implement this feature just like the echo command. You don't need abstraction. You can just duplicate the command architecture from other commands
![Screen Shot 2022-10-20 at 16 49 33](https://user-images.githubusercontent.com/3696314/196967201-286a96e3-48cb-4302-a1c0-c4ee4843c91d.png)

**Comment by abdrd** [comment]
I had implemented it like this: 
```go
const (
	_HELP = `Usage: whoami [OPTION]
Print  the  user name associated with the current effective user ID.  
Same as id -un.

	--help, -h 		display this help and exit
	--version, -v 	output version information and exit		
`
	_VERSION = "0.0.1"
	_CMDNAME = "whoami"
)

func whoami(args []string) error {
	argsLen := len(args)
	switch argsLen {
	case 1:
		// only program name
		u, err := user.Current()
		if err != nil {
			return err
		}
		fmt.Println(u.Username)
	case 2:
		arg := args[1]
		switch arg {
		case "--help", "-h":
			fmt.Print(_HELP)
		case "--version", "-v":
			fmt.Printf("whoami - version %s\n", _VERSION)
		default:
			return fmt.Errorf("unknown command '%s'; pass '--help', or '-h' for help information", arg)
		}
	default:
		return fmt.Errorf("excessive number of arguments (%d)", argsLen)
	}
	return nil
}

func Run(args []string) error {
	return whoami(args)
}

/// TESTS

```

OK then... Good luck

**Comment by abdrd** [comment]
You can deassign me

**Comment by yakuter** [comment]
@betelgeuse-7 I would like you to complete the PR if it is OK with you. You could have just used the lines you wrote with a command template like [echo here](https://github.com/yakuter/gol/blob/main/commands/echo/echo.go)

```
u, err := user.Current()
if err != nil {
	return err
}
fmt.Println(u.Username)
```

**Comment by yakuter** [comment]
Alright, I deassigned you.

**Comment by yakuter** [comment]
Closed by #18

**Comment by abdrd** [comment]
Sorry for wasting your time. I hopped into this without actually reading the code-base. 
Again, sorry...
